pkg/output/output_setup.go

```diff
@@ -110,3 +112,34 @@ func ParseBoolFlag(long string, short string) bool {
 func ColorsEnabled() bool {
 	return os.Getenv("NO_COLOR") == "" || os.Getenv("NO_COLOR") == "0"
 }
+
+// WaitTimer tracks elapsed time for wait operations with inline output
+type WaitTimer struct {
+	startTime time.Time
+}
+
+// StartWait prints a waiting message (without newline) and returns a WaitTimer
+// to track elapsed time. Call Complete() when the operation finishes.
+// In JSON mode, no message is printed.
+func StartWait(message string) *WaitTimer {
+	if !JSONOutput {
+		_, _ = fmt.Fprintf(os.Stdout, "%s...", message)
+	}
+	return &WaitTimer{startTime: time.Now()}
+}
+
+// Complete prints the elapsed time on the same line as the wait message.
+// If err is non-nil, just prints a newline. If extra is provided, it's appended after the time.
+func (w *WaitTimer) Complete(err error, extra string) time.Duration {
+	elapsed := time.Since(w.startTime)
+	if !JSONOutput {
+		if err != nil {
+			_, _ = fmt.Fprintf(os.Stdout, "\n")
+		} else if extra != "" {
+			_, _ = fmt.Fprintf(os.Stdout, " ready in %.1fs, %s\n", elapsed.Seconds(), extra)
```

Comment:
I'm not sure that `extra` is needed here, because we say the same thing twice:

```
WARNING: Router count mismatch: 0 loaded, 6 expected
```

<img width="1316" height="194" alt="image" src="https://github.com/user-attachments/assets/bdb3b9de-7445-414b-b226-8a9fc20e6f3c" />

---

```
WARNING: Detected 1 configuration error(s) - check 'docker logs ddev-router' for details
```

<img width="806" height="293" alt="image" src="https://github.com/user-attachments/assets/3fc2a10e-4a52-4ab3-a569-f37b1f6547c9" />

---

And seeing `OK: http://:10999/ping` from `traefik healthcheck --ping` may be confusing to our users:

<img width="887" height="108" alt="image" src="https://github.com/user-attachments/assets/ab2cff5b-c17d-49f1-87f2-6ea04a303b23" />

Reply:
Thanks, it's much improved with that suggestion.